scripts/export.js

```diff
@@ -1,115 +1,122 @@
-var Promise = require('bluebird');
-var program = require('commander');
-var pgClient = require('pg').Client;
-var Tar = require('tar-stream');
-var colors = require('colors');
-var fs = Promise.promisifyAll(require('fs'));
-var path = require('path');
-
-var client;
-var projectID;
-var project;
-var destination;
+"use strict";
+
+const Promise = require(`bluebird`);
+const program = require(`commander`);
+const PgClient = require(`pg`).Client;
+const Tar = require(`tar-stream`);
+const colors = require(`colors`);
+const fs = Promise.promisifyAll(require(`fs`));
+const path = require(`path`);
+
+let client;
+let projectID;
+let project;
+let destination;
 
 colors.setTheme({
-  silly: 'rainbow',
-  input: 'grey',
-  verbose: 'cyan',
-  prompt: 'grey',
-  info: 'green',
-  data: 'grey',
-  help: 'cyan',
-  warn: 'yellow',
-  debug: 'blue',
-  error: 'red'
+  silly: `rainbow`,
+  input: `grey`,
+  verbose: `cyan`,
+  prompt: `grey`,
+  info: `green`,
+  data: `grey`,
+  help: `cyan`,
+  warn: `yellow`,
+  debug: `blue`,
+  error: `red`
 });
 
 function getMessage() {
-  var message = '';
+  var message = ``;
   Array.prototype.forEach.call(arguments, function(arg) {
     if(arg) {
-      message += arg + ' ';
+      message += arg + ` `;
     }
   });
   message = message.trim();
   return message;
 }
 
-var log = {
-  error: function() {
-    var message = getMessage.apply(null, arguments);
-    console.error(('ERROR: ' + message).error);
-    console.error('\n\n');
+class Log {
+  static error() {
+    const message = getMessage.apply(null, arguments);
```

Comment:
apply in ES6 is highly questionable - what are you trying to do here? `getMessage(...arguments)` should do the trick already